Libs/Gst/Structs/Message.cs

```diff
@@ -19,7 +19,7 @@ public partial struct Message
         
         public Gst.Object Src
         {
-            get => GObject.Object.WrapHandle<Gst.Object>(src);
+            get => GObject.Object.WrapHandle<Gst.Object>(src, false);
```

Comment:
This one will need some investigation. It's probably better to drop the `Src` field altogether and just force the user to go through `Message.GetStructure` because we at least have introspection data for it. Let's not block on this for now though.

Reply:
Okay :+1: I don't have any opinion here as I do not have any experience with the GStreamer API. But like you said as this is probably fine tuning let's discuss this later. I added a TODO flag to the property.